src/process_dialog.rs

```diff
@@ -82,6 +82,12 @@ pub fn create_process_dialog(process: &sysinfo::Process, window: &gtk::Window,
     vertical_layout.pack_start(&close_button, false, true, 0);
     vertical_layout.set_spacing(10);
     area.pack_start(&vertical_layout, true, true, 0);
+    // To silence the annying warning:
+    // "(.:2257): Gtk-WARNING **: Allocating size to GtkWindow 0x7f8a31038290 without
+    // calling gtk_widget_get_preferred_width/height(). How does the code know the size to
+    // allocate?"
+    popup.get_preferred_width();
+    popup.set_size_request(500, 700);
```

Comment:
Maybe you actually want to make use of the preferred width/height you get back from the popup? :) The warning is probably there for a reason

Reply:
In certain cases, it makes sense but in others, the window will get too big because of the amount of information. Like this, it's always the same. :)